### PR TITLE
PWN-4394 - Main Screen Buttons Added

### PR DESCRIPTION
**Pull request by kabramovp2p**
### Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-4394

### Description of Work

Redesigned home action buttons

### Screenshots 

Android Emulator - Pixel_5_API_30:5554 2022-08-08 15-47-33

### CODE REVIEW

**Review by kuznetsov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/home/ui/main/HomeFragment.kt [hunk 1]
```diff
@@ -125,18 +121,34 @@ class HomeFragment :
...
+            textView.setText(R.string.main_buy)
+            imageButton.setImageResource(R.drawable.ic_plus)
+            imageButton.setOnClickListener {
+                presenter.onBuyClicked()
+            }
         }
-        onReceiveItemClickListener = {
-            replaceFragment(ReceiveSolanaFragment.create(token = null))
+        actionReceiveView.apply {
+            textView.setText(R.string.main_receive)
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/home/ui/main/HomeFragment.kt:133]: main -> home

File: app/src/main/res/layout/fragment_home.xml [hunk 2]
```diff
@@ -20,17 +20,31 @@
...
+            <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginTop="@dimen/home_toolbar_height"
-                app:layout_collapseMode="none" />
+                android:orientation="vertical"
+                app:layout_collapseMode="none">
+
+                <include
+                    android:id="@+id/balance"
```
Comment (gslevinkov) [inline app/src/main/res/layout/fragment_home.xml:33]: layout_balance
/
view_balance

File: app/src/main/res/layout/item_main_button.xml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (gslevinkov) [inline app/src/main/res/layout/item_main_button.xml:1]: item_home_button

File: app/src/main/res/layout/item_main_button.xml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        android:id="@+id/imageButton"
+        android:layout_width="56dp"
+        android:layout_height="56dp"
+        android:background="@drawable/bg_circle_button"
+        android:foreground="@drawable/ripple_button_circle"
+        android:importantForAccessibility="no"
+        android:src="@drawable/ic_plus" />
+
+    <TextView
+        android:id="@+id/textView"
```
Comment (gslevinkov) [inline app/src/main/res/layout/item_main_button.xml:23]: text_view_button_title

File: app/src/main/res/layout/item_main_button.xml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        style="@style/UiKit.TextAppearance.SemiBold.Text4"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
+        android:maxLines="1"
+        android:textColor="@color/textIconPrimary"
```
Comment (gslevinkov) [inline app/src/main/res/layout/item_main_button.xml:29]: text color should be from uiKit

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/home/ui/main/HomeFragment.kt (hunk 1 above)
Comment (kabramovp2p) [reply app/src/main/java/org/p2p/wallet/home/ui/main/HomeFragment.kt]: no
main buy and receive alreade there
need to rename in future

**Pull request merged**